### PR TITLE
Employee add/list/update/delete

### PR DESCRIPTION
**Pull request by wrobelmichal**

### CODE REVIEW

File: easywarehouse/products/models.py [hunk 1]
```diff
@@ -16,6 +17,11 @@ class Product(models.Model):
     tags = ArrayField(models.CharField(max_length=50), blank=True)
     description = models.TextField(default="")
     main_image_url = models.URLField(null=True, blank=True)
+    ean = models.CharField(
+        max_length=13,
+        validators=[RegexValidator(regex="[0-9]{13}", message="EAN can contain only numbers")],
+        default="0000000000000",
```
Comment (songofroland) [inline easywarehouse/products/models.py:23]: let's rather make it nullable than default zeros

File: easywarehouse/static/style.css [hunk 2]
```diff
@@ -22,6 +22,9 @@ body {
     background-color: var(--far-background);
     font-family: Arial, Helvetica, sans-serif;
 }
+main{
```
Comment (songofroland) [inline easywarehouse/static/style.css:25]: insert empty line

File: easywarehouse/templates/categories/delete.html [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
-    <input type="submit" value="Confirm">
-</form>
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+    <form method="post">
+        {% csrf_token %}
+        <p>Are you sure you want to delete "{{ object }}"?</p>
+        <input type="submit" value="Confirm">
```
Comment (songofroland) [inline easywarehouse/templates/categories/delete.html:8]: There should be cancel button, but u can add it in next MR

File: easywarehouse/templates/images/list.html [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
-  <ul>
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+  <link rel="stylesheet" type="text/css" href="{% static 'list.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'images.css' %}">
+
+ <h1>Images</h1>
     {% for image in object_list %}
```
Comment (songofroland) [inline easywarehouse/templates/images/list.html:9]: let's make it HTML list too (ul & li)

**Review by songofroland — CHANGES_REQUESTED** [review]
Look good, did not check it locally tho

**Comment by songofroland** [comment]
Don't forget to add missing things in the next MR

**Review by songofroland — APPROVED** [review]

File: easywarehouse/static/images.css [hunk 5]
```diff
@@ -0,0 +1,4 @@
+img{
+    height: 50vh;
+    max-width: 80vh;
+}
```
Comment (wilmhit) [inline easywarehouse/static/images.css:4]: EOF

**Review by wilmhit — APPROVED** [review]

**Comment by wilmhit** [comment]
templates/categories/... => templates/employee/categories/...
templates/images/... => templates/employee/images/...
templates/products/... => templates/employee/products/...

**Pull request merged**